Scripts/ScreenshotUtils/ScreenshotWindow.cs

```diff
@@ -63,20 +81,20 @@ private void OnGUI()
 				camera = renderCamera;
 			}
 
-			var renderTexture = camera.targetTexture;
-
 			fileFileFormat = (FileFormat)EditorGUILayout.EnumPopup("Format", fileFileFormat);
 			if (fileFileFormat == FileFormat.JPEG)
 			{
 				RenderJPEGOptions();
 			}
 
 			textureFormat = (TextureFormat)EditorGUILayout.EnumPopup("Texture Format", textureFormat);
+			renderTextureFormat = (RenderTextureFormat)EditorGUILayout.EnumPopup("Render Texture Format", renderTextureFormat);
 
 			EditorGUILayout.BeginHorizontal();
 			width = EditorGUILayout.IntField("Width", width);
 			height = EditorGUILayout.IntField("Height", height);
 			EditorGUILayout.EndHorizontal();
+			antiAliasing = EditorGUILayout.IntField("Anti Aliasing (0, 1, 2, 4, 8)", antiAliasing);
```

Comment:
Has this been tested with 0? The docs state 
"Number of antialiasing samples to store in the texture. Valid values are 1, 2, 4, and 8. Throws an exception if any other value is passed."

Reply:
Nope. I previously tested on PS4 (with something like  `antiAlising++`) and it didn't throw. And I can't be arsed to write an enum and convert its value to it.

However, I believe an enum should be the way to go. I'll get this fixed.